app/main.py

```diff
@@ -1,3 +1,20 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Callable:
+    _cache = dict()
```

Comment:
Since your function is called c cache, you use the _cache variable as a dictionary, it is better to rename it to cache_dict, the underscore in python is used for encapsulation, you will learn about this later